exercises/high-scores/example.js

```diff
@@ -0,0 +1,27 @@
+export class HighScores {
+  constructor(scores) {
+    this.scores = scores;
+  }
+
+  latest() {
```

Comment:
I wonder if we should take this exercise as an opportunity to allow [getters](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/get) or object properties rather than requiring functions to return these values.

I ask because this is a feature of ES 2015 and beyond, and we're trying to make a point of exposing users to them.

@PakkuDon what do you think about that?

Reply:
@matthewmorgan TIL, thanks for the article! 
Switched to getters in d14d8c3